apps/mobile/src/libs/components/text/styles.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+	text: {
+		color: "#000000",
+		fontSize: 16,
```

Comment:
As soon as we have multiple fonts, define the default fontFamily for Text component.
Also, let's pass color as a prop with some default value.

Reply:
I want to clarify how I can add fonts to the project. Can I add fonts by creating react-native.config.js at the project's root or in any other way?

Reply:
Yes, you can try this way. You need to ensure that in your android/app/src/main/res folder, the 'font' directory is created during the build.

Reply:
Thanks for the review. I've updated the code according to your comment. Could you please take a look?